core/src/main/java/de/danoeh/antennapod/core/feed/FeedItem.java

```diff
@@ -388,6 +391,22 @@ public boolean hasChapters() {
         return hasChapters;
     }
 
+    public void setAutoDownload(boolean autoDownload) {
+        this.autoDownload = autoDownload;
+    }
+
+    public boolean getAutoDownload() {
+        return this.autoDownload;
+    }
+
+    public boolean isAutoDownloadable() {
+        return this.hasMedia() &&
+                false == this.getMedia().isPlaying() &&
```

Comment:
I only recently started doing that myself. It really only makes sense if you compare booleans, for any other object it does not (the compiler will catch those). If you google yoda conditions, a lot of people complain about reversing the order - `5 >= foo.bar()` or `null == foo.bar()` should really be avoided.